### PR TITLE
feat: `cdktf` uses remote content

### PR DESCRIPTION
**Pull request by thiskevinwang**
# Description

This updates the `Dockerfile` to work from arm64 machines (M1 MacBook)

This removes the `terraform-cdk` related symlinks and submodule

---

Blocks https://github.com/hashicorp/terraform-cdk/pull/1608

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,4 +51,9 @@ website-local:
...
 
 .DEFAULT_GOAL := website
-.PHONY: build build-image website website-local
\ No newline at end of file
+.PHONY: build build-image website website-local
+
+test-build:
+	docker rmi hashicorp-terraform-website-local || true
+	docker rmi hashicorp-terraform-website-local:latest || true
+	docker build -t hashicorp-terraform-website-local:latest .
```
Comment (thiskevinwang) [inline Makefile:59]:
```suggestion
.PHONY: build build-image website website-local
```

File: package.json [hunk 2]
```diff
@@ -66,6 +66,8 @@
   "main": "index.js",
   "scripts": {
     "build": "next build",
+    "build:deploy-preview": "npx ts-node --transpile-only ./scripts/content-repo-preview/build.ts",
+    "postbuild": "echo 'at postbuild' && echo $(pwd)",
```
Comment (BRKalow) [inline package.json:70]:
```suggestion
```

File: pages/intro/[[...page]].jsx [hunk 3]
```diff
@@ -31,6 +31,7 @@ export async function getStaticPaths() {
   const paths = await generateStaticPaths({
     navDataFile: NAV_DATA,
     localContentDir: CONTENT_DIR,
+    product: PRODUCT,
```
Comment (thiskevinwang) [inline pages/intro/[[...page]].jsx:34]: Needed to add these as a result of a breaking change in `react-docs-page\

**Comment by thiskevinwang** [comment]
Latest deploy preview
https://terraform-website-jmme03ubw-hashicorp.vercel.app/cdktf

**Review by BRKalow — APPROVED** [review]
👏

**Pull request merged**